src/js/views/patients/schedule/layout.hbs

```diff
@@ -0,0 +1,9 @@
+<div class="list-page__header">
```

Comment:
There's a lot of classes in here that aren't used for anything.  Can we clean that up?  Also if it can be avoided I'd like to not use a class for just cypress.  and prefer`.js-` or region selectors over style ones